src/features/shared-link-settings-modal/SharedLinkSettingsModal.js

```diff
@@ -310,30 +311,52 @@ class SharedLinkSettingsModal extends Component {
     renderAllowDownloadSection() {
         const {
             canChangeDownload,
+            item,
             directLink,
             directLinkInputProps,
             downloadCheckboxProps,
             isDirectLinkAvailable,
             isDirectLinkUnavailableDueToDownloadSettings,
+            isDirectLinkUnavailableDueToAccessPolicy,
             isDownloadAvailable,
         } = this.props;
         const { isDownloadEnabled } = this.state;
-
+        const { classification } = item;
         return (
             <AllowDownloadSection
                 canChangeDownload={canChangeDownload}
+                classification={classification}
                 directLink={directLink}
                 directLinkInputProps={directLinkInputProps}
                 downloadCheckboxProps={downloadCheckboxProps}
                 isDirectLinkAvailable={isDirectLinkAvailable}
                 isDirectLinkUnavailableDueToDownloadSettings={isDirectLinkUnavailableDueToDownloadSettings}
+                isDirectLinkUnavailableDueToAccessPolicy={isDirectLinkUnavailableDueToAccessPolicy}
                 isDownloadAvailable={isDownloadAvailable}
                 isDownloadEnabled={isDownloadEnabled}
                 onChange={this.onAllowDownloadChange}
             />
         );
     }
 
+    renderModalTitle() {
+        const { item } = this.props;
+        const { bannerPolicy, classification } = item;
+        const title = <FormattedMessage {...messages.modalTitle} />;
+
+        return (
+            <span className="bdl-SharedLinkSettingsModalTitle">
+                {title}
+                <Classification
+                    advisoryMessage={bannerPolicy ? bannerPolicy.body : undefined}
+                    messageStyle="tooltip"
+                    name={classification}
+                    className="bdl-SharedLinkSettingsModalTitle-classification"
```

Comment:
In a similar manner, `bdl-SharedLinkSettingsModal-titleClassification` or perhaps just `bdl-SharedLinkSettingsModal-classification`